### ISSUE TITLE
Bug: Map location being updated to default, even though geolocation is disabled

### ISSUE THREAD

**Issue by paulnagle** [bmlt-enabled/bmlt-workflow#116]
Hi!
In the BMLT Workflow settings, geolocation being disabled is being noted
```
This plugin honours the BMLT Root Server Auto Geocoding settings. The $auto_geocoding_enabled setting is set to false;

Meeting submissions will not be automatically geocoded on save
```
However, when I make a change to a meeting, the meetings location is being updated to the default location settings lat,long that are set in the BMLT.
i.e. In my BMLT I have the following settings in my auto-config.inc.php file
```
$search_spec_map_center = array ( 'longitude' => -5.985280, 'latitude' => 53.364817, 'zoom' => 12 ); // This is the default map location for new meetings.
$auto_geocoding_enabled = false;
```
The BMLT Workflow app is updating every meeting that it changes to have the default lat/long that are set in $search_spec_map_center.

What I need is for the BMLT Workflow app to 
- not change the lat/long of any meeting that is updated
- Set the lat/long of any new meetings to the defaults set in $search_spec_map_center.

Thanks!

**Comment by nigel-bmlt** [comment]
Hi Paul,

Could you let me know the bmlt version you are using, and also could you screenshot me a meeting from within the BMLT console that has its lat long updated incorrectly, before and after the update is made via the workflow plugin?

I've just tried to reproduce this on bmlt2x and had no luck, so your examples would help track it down for me.

**Comment by paulnagle** [comment]
Sure, BMLT Version is 2.16.6
Will track down some screenshots shortly...

**Comment by paulnagle** [comment]
Here is a quick screen recording of this happening.... I look at the location of the meeting in BMLT first, then I process a change to the meeting listing via BMLT Workflow, then I go back to the BMLT, refresh the page, and look at the meeting again in BMLT. You can see that the co-ordinates change to the default setting.


https://user-images.githubusercontent.com/3714999/210977136-f46a2ee0-94a3-4432-8b9c-aa630797a9c1.mp4

**Comment by nigel-bmlt** [comment]
fixed in 1.0.16
